x-pack/plugins/snapshot_restore/public/application/sections/home/policy_list/policy_details/tabs/tab_summary.tsx

```diff
@@ -86,7 +86,7 @@ export const TabSummary: React.FunctionComponent<Props> = ({ policy }) => {
                   description={i18n.translate(
                     'xpack.snapshotRestore.policyDetails.snapshotsTakenStat',
                     {
-                      defaultMessage: 'Snapshots',
+                      defaultMessage: 'Total snapshots taken',
```

Comment:
```suggestion
                      defaultMessage: 'Snapshots taken',
```

I don't think _total_ or _number of_ is necessary in the context, and leading with Snapshot emphasizes that these are snapshot stats.

Reply:
Thank you for the review.